File: 0392-is-subsequence/0392-is-subsequence.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isSubsequence = function(s, t) {
    let ind1 =0;
    let ind2 =0;

    while( ind1<s.length && ind2<t.length){
        if(s[ind1]===t[ind2]){
            ind1++;
        }
        ind2++;

    }

    return ind1 === s.length
    
    
};

File: 0392-is-subsequence/0392-is-subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        count = 0

        for i in t:
            if count<len(s) and i == s[count]:
                count +=1
        if (count == len(s)):
            return True
        else:
            return False
